data/json/recipes/armor/torso.json

```
@@ -87,7 +87,7 @@
     "autolearn": true,
     "book_learn": [ [ "recipe_melee", 2 ] ],
     "qualities": [ { "id": "CUT", "level": 1 } ],
-    "components": [ [ [ "paper", 300 ] ], [ [ "duct_tape", 150 ] ] ]
+    "components": [ [ [ "paper", 1200 ] ], [ [ "duct_tape", 150 ] ] ]
```

```suggestion
    "components": [ [ [ "paper", 1200 ] ], [ [ "duct_tape", 150 ], [ "cordage", 8, "LIST" ], [ "cordage_superior", 2, "LIST" ] ] ]
```
This lets you tie it on if you don't have duct tape.

---

I dig it, good addition.